client/src/components/Item/index.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+
+class Item extends Component {
+  constructor(props) {
+    super(props);
+    this.clickHandler = this.clickHandler.bind(this);
+  }
+
+  //invoke
+  clickHandler() {
+    console.log('this.props.id is :', this.props.id)
+    this.props.cancelItem(this.props.id)
+  }
+
+  render() { 
+    const name = this.props.name;
```

Comment:
Use destructuring to take all this props out of the props object in a row:
`const { name, category, id } = this.props;`